src/background/agent.js

```diff
@@ -41,6 +41,8 @@ const Contents = {
 const Logs = {
   update: (log) =>
     requests.put(`/logs/${log.id}`, { log }),
+  delete: (log) =>
+    requests.del(`/logs/${log.id}`, { log }),
```

Comment:
since we only need the `id` here, we can clean this up and just accept id and pass that in directly

```suggestion
  delete: id =>
    requests.del(`/logs/${id}`),
```